server/models/profileModel.js

```diff
@@ -0,0 +1,35 @@
+const db = require("./conn");
+
+class Profile {
+  constructor(
+    email,
+    password,
+    first_name,
+    last_name,
+    github_url,
+    linkedin_url,
+    user_type,
+    bootcamp_name
+  ) {
+    this.email = email;
+    this.password = password;
+    this.first_name = first_name;
+    this.last_name = last_name;
+    this.github_url = github_url;
+    this.linkedin_url = linkedin_url;
+    this.user_type = user_type;
```

Comment:
all of the same logic is already in the userModel. You can just add the function getbyID into the userModel.

Reply:
I'm not seeing getbyId in the userModel. Maybe it's in a different branch on your local environment?